### PR TITLE
VMO-3576: Error when dragging a block after deleting one expression

### PR DESCRIPTION
**Pull request by poojahpatil90**

### CODE REVIEW

**Comment by safydy-r** [comment]
Wow, this is very clean @poojahpatil90 . Thank :100:

**Review by safydy-r — APPROVED** [review]

**Comment by safydy-r** [comment]
Pls update from master and check TS errors.
After, you should not have this `node-sass` related build error

File: src/components/common/PlainDraggable.vue [hunk 1]
```diff
@@ -90,6 +90,7 @@ export default {
   },
 
   destroyed() {
+    this.$emit('destroyed')
```
Comment (bzabos) [inline src/components/common/PlainDraggable.vue:93]: Can you add `{ draggable }` as payload to this event?

**Review by bzabos — APPROVED** [review]
This is perfect @poojahpatil90! Thanks! :)

File: src/components/common/PlainDraggable.vue (hunk 1 above)
Comment (poojahpatil90) [reply src/components/common/PlainDraggable.vue:93]: Sure, I will @bzabos!  It was not required in this context so I hadn't added.

**Pull request merged**